pkg/tasks/queue/pgqueue.go

```diff
@@ -228,7 +229,6 @@ func NewPgxPool(ctx context.Context, url string) (*pgxpool.Pool, error) {
 	// Allow for context cancellation to release the pool
 	go func() {
 		<-ctx.Done()
-		pool.Close()
 	}()
```

Comment:
this entire go routine can be removed because it's only used for closing the pool

Reply:
removed 🫡